### DISCUSSION TITLE
Handlebars registerHelper

### DISCUSSION THREAD

**Discussion by toog** [fbaligand/kibana-enhanced-table#275] [Q&A]
Hello

Is it possible to register helper for Handlebars in template ?
I try but not with success.

![image](https://user-images.githubusercontent.com/40697794/171212541-8071cf25-814b-4123-b373-396ffc4bc0e7.png)

the error :

```
description: undefined​
endLineNumber: undefined​
fileName: "../built_assets/dlls/vendors_0.bundle.dll.js"​
lineNumber: 81​
message: "Missing helper: \"loud\""​
name: "Error"​
number: undefined
```

**Comment by fbaligand** [comment]
Sorry, it is not possible.

**Comment by fbaligand** [comment]
But you can put “helper” code in Formula
